x-pack/plugins/spaces/server/saved_objects/mappings.ts

```diff
@@ -38,3 +38,46 @@ export const SpacesSavedObjectMappings = deepFreeze({
     },
   },
 });
+
+export const SpacesTelemetryMappings = deepFreeze({
+  properties: {
+    apiCalls: {
+      type: 'object',
+      properties: {
+        copySavedObjects: {
+          type: 'object',
+          properties: {
+            total: { type: 'long' },
+            createNewCopies: {
+              type: 'object',
+              properties: {
+                enabled: { type: 'long' },
```

Comment:
```suggestion
                enabled_total: { type: 'long' },
```
Reason: the remote telemetry services' indices contain many fields named `enabled` and `disabled` and those are typically booleans. We also have plugins that reference config settings named `enabled/disabled`, also referring to booleans.

Reply:
Instead of "createNewCopies.enabled", "createNewCopies.disabled", etc...
I changed these to "createNewCopiesEnabled.yes", "createNewCopiesEnabled.no", etc.

I feel like that's less likely to be conflated with existing telemetry fields. In addition, this allows our naming of usage stats fields to be a bit more consistent, since we already had "allTypes.yes" and "allTypes.no".

Happy to defer to @elastic/kibana-core if they like these mapping names or want to change them to something else.